iconservice/database/batch.py

```diff
@@ -159,8 +216,25 @@ def __init__(self, block: Optional['Block'] = None):
     def __setitem__(self, key, value):
         raise AccessDeniedException("Can not set data on block batch directly.")
 
+    def to_list(self) -> list:
+        """
+        Return list of key, value for Debugging
+        :return:
+        """
+        new_list: list = []
+        for key, val in self.items():
+            _dict = {"key": key}
+            _dict.update(val.to_dict(to_camel_case))
+            new_list.append(_dict)
+        return new_list
+
     def update(self, tx_batch: 'TransactionBatch', **kwargs):
```

Comment:
If the same key has been put on block_batch, pop previous value for extract the tx index